### ISSUE TITLE
Error when playing music from loaded png

### ISSUE THREAD

**Issue by felipebueno** [picolove/picolove#11]
Hi, @ftsf  ! @felipebueno from the PICO-8 BBS here.

I saw the video where you play Celeste on PicoLove at 60fps and it's preatty neat. So I cloned your repo and tried to run it here but it's throwing an error: `Error: main.lua:1227: attempt to index local 'm' (a nil value)`
![](http://i.imgur.com/eSfAqN5.png)

**Comment by felipebueno** [comment]
Hey, @ftsf you can ignore this issue. I realized the problem... I was trying to run the png file, not the p8.

It's working now :smile:

**Comment by felipebueno** [comment]
How do you regulate the framerate at 60fps? It runs too fast hahah

**Comment by ftsf** [comment]
I don't think this issue is resolved, there is a bug with music when loading from PNG =)
I'll update the issue name.

**Comment by ftsf** [comment]
@felipebueno it requires changes to the game to operate correctly at 60fps.

http://static.impbox.net/celeste60fps.p8

**Comment by MarionParker** [comment]
I realized the problem... I was trying to run the png file
https://ringtonedownload.in/

**Comment by mp3tomato** [comment]
here [https://mp3tomato.com/](https://mp3tomato.com/)
